VideoLocker/res/layout/row_course_outline_list.xml

```diff
@@ -104,6 +104,7 @@
             android:background="@android:color/transparent"
             android:paddingEnd="5dp"
             android:paddingRight="5dp"
```

Comment:
The padding and margin dimensions are quite weird here; this `LinearLayout` has both padding and margin for right/end, and the right-most view has it's own 1 DIP right/end padding. But I guess we can leave these alone for now.